src/doc/style-guide/src/README.md

```diff
@@ -28,7 +28,7 @@ formatting, typically by using a formatting tool's default settings.
 * Each level of indentation must be four spaces (that is, all indentation
   outside of string literals and comments must be a multiple of four).
 * The maximum width for a line is 100 characters.
-* A tool should be configurable for all three of these variables.
+* A tool may choose to make some of these configurable.
```

Comment:
We may just want to drop this altogether.

Reply:
@compiler-errors I agree entirely, but I wanted this PR to avoid making any semantic changes. Happy to drop it in a subsequent PR.